frontends/web/src/components/help/help.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, h, RenderableProps } from 'preact';
+import * as style from './help.css';
+
+interface HelpType {
+    text: string;
+    position: string;
+}
+
+interface State {
+    show: boolean;
+}
+
+type Props = HelpType;
+
+class Help extends Component<Props, State> {
+    constructor(props) {
+        super(props);
+        this.state = {
+            show: false,
+        };
+    }
+
+    private showText = () => {
+        this.setState({ show: true }, () => {
+            this.registerClick();
+        });
+    }
+
+    private registerClick = () => {
+        document.addEventListener('click', this.handleClick);
+    }
+
+    private unregisterClick = () => {
+        document.removeEventListener('click', this.handleClick);
+    }
+
+    private handleClick = (e: Event) => {
+        const classList = (e.target as HTMLElement).classList;
+        if (!classList.contains(style.box) && !classList.contains(style.container)) {
+            e.preventDefault();
+            this.setState({ show: false }, () => {
+                this.unregisterClick();
+            });
+        }
+    }
+
+    public render(
+        { position, text }: RenderableProps<Props>,
+        { show }: State,
+    ) {
+        return (
+            <a className={style.container} onClick={this.showText}>?
```

Comment:
Hmmm, generally I would expect tooltips to show up on `hover`, not on click. Isn't that the idea of a tooltip?

Reply:
ideally both, and also on focus + tabindex